src/main/resources/config/application-hades.yml

```diff
@@ -0,0 +1,8 @@
+artemis:
+    hades:
```

Comment:
All other CI solutions in Artemis put their config under `continuous_integration`.
We should stay consistent in how we do the configs. 
(Though I do agree that having every VC & CI config like this would be more beneficial, but that would have to be changed for all VC & CI implementations in a separate PR and would be a breaking change)